packages/client/hmi-client/src/components/documents/tera-document-asset.vue

```diff
@@ -146,6 +146,11 @@ enum DocumentView {
 	NOT_FOUND = 'Not found'
 }
 
+enum AssetType {
+	FIGURE = 'figure',
+	TABLE = 'table',
+	EQUATION = 'equation'
+}
```

Comment:
I think those should come from the back-end with the Type Generator:
https://github.com/DARPA-ASKEM/terarium/blob/main/packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/document/DocumentExtraction.java#L15-L17

Reply:
sure ill look into it!

Reply:
should be good now